mission_control/navigator_missions/navigator_singleton/pose_editor.py

```diff
@@ -203,7 +204,7 @@ def to_lat_long(self, lat, lon, alt=0):
     def circle_point(self, point, radius, granularity=8, theta_offset=0):
```

Comment:
Eh, the idea is it's like `move.forward`.  So instead of moving forward you're circling a point not a circling point gen. But the circle code will change when lqrrt circling works